enzyme/test/Enzyme/ReverseMode/writeonlyretjl.ll

```diff
@@ -0,0 +1,93 @@
+; RUN: %opt < %s %loadEnzyme -enzyme -enzyme-preopt=false -mem2reg -instsimplify -simplifycfg -S -enzyme-zero-cache=1 | FileCheck -check-prefixes CHECK,ZERO %s
+
+declare dso_local double @__enzyme_autodiff(i8*, double)
+
+define void @subsq(double addrspace(10)* writeonly nocapture %out, double %x) {
+entry:
+  %mul = fmul double %x, %x
+  store double %mul, double addrspace(10)* %out, align 8
+  ret void
+}
+
+declare {}*** @julia.get_pgcstack()
+declare {} addrspace(10)* @jl_gc_alloc_typed(i8*, i64, {} addrspace(10)*)
+
+define double @mid(double %x) {
+  %pg = call {}*** @julia.get_pgcstack() "enzyme_inactive" readnone "enzyme_shouldrecompute"
+  %p3 = bitcast {}*** %pg to {}**
+  %p4 = getelementptr inbounds {}*, {}** %p3, i64 -12
+  %p5 = getelementptr inbounds {}*, {}** %p4, i64 14
+  %p6 = bitcast {}** %p5 to i8**
+  %p7 = load i8*, i8** %p6, align 8
+  %al = call noalias nonnull dereferenceable(8) dereferenceable_or_null(8) {} addrspace(10)* @jl_gc_alloc_typed(i8* %p7, i64 8, {} addrspace(10)* addrspacecast ({}* inttoptr (i64 139806792221568 to {}*) to {} addrspace(10)*)), !enzyme_fromstack !{i64 8}
```

Comment:
Note @vchuravy right now in Enzyme.jl we will always emit this fallback -- however we only actually need to do so if its not a constant size. I won't change this ll test, but FYI we do so in Enzyme.jl